vault-auth/install.sls

```diff
@@ -2,6 +2,7 @@ install_python_dependencies:
   pip.installed:
     - pkgs:
       - hvac
+      - hvac[parser]
```

Comment:
Curious. What led to this change?

Reply:
That was troubleshooting the missing pkgs error. I think I can safely remove it.